content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -0,0 +1,170 @@
+
+```bash
+dataflow:>app register --name myTask --type task --uri https://github.com/tzolov/task-demo-metrics/raw/master/apps/task-demo-metrics-0.0.1-SNAPSHOT.jar
+
+dataflow:>task create --name task1 --definition "myTask"
+dataflow:>task create --name task2 --definition "myTask"
+```
+
+Launch the tasks several times:
+
+```bash
+dataflow:>task launch --name task1
+dataflow:>task launch --name task2
+```
+
+In the [DataFlow task execution UI](http://localhost:9393/dashboard/#/tasks/executions) you should see list like this:
+![SCDF Task Execution](images/SCDF-metrics-task-execution-view.png)
+
+You should see dashboards similar to those shown in the following image:
+
+![SCDF Task Grafana InfluxDB](images/SCDF-metrics-grafana-task.png)
+
+## Cloud Foundry
+
+This section describes how to set up InfluxDB for Cloud Foundry.
+
+### InfluxDB
+
+You can follow the general [Manifest based installation on Cloud Foundry](http://localhost:8000/docs/installation/cloudfoundry/cf-cli/#manifest-based-installation-on-cloud-foundry) instructions for installing `Skipper` and `DataFlow` on Cloud Foundry.
+To enabling the Task metrics integration you need to extend the [DataFlow manifest](http://localhost:8000/docs/installation/cloudfoundry/cf-cli/#installing-using-a-manifest) with following `SPRING_APPLICATION_JSON` variable:
+
+```json
+{
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.enabled": true,
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.db": "yourinfluxdb",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.autoCreateDb": false,
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.uri": "https://your-influx-uri:port",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.userName": "influxusername",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.password": "******",
+  "spring.cloud.dataflow.grafana-info.url": "https://yor-grafana-uri:443"
+}
+```
+
+Check the [Influx Actuator properties](https://docs.spring.io/spring-boot/docs/2.2.0.M4/reference/html/#actuator-properties) for further details about the `management.metrics.export.influx.XXX` properties.
+
+Complete example of a DataFlow manifest that enables metrics collection for both Stream and Tasks would look like this:
+
+```yml
+---
+applications:
+- name: tzolov-data-flow-server
+  host: tzolov-data-flow-server
+  memory: 2G
+  disk_quota: 2G
+  instances: 1
+  path: ./spring-cloud-dataflow-server-2.2.0.M1.jar
+  env:
+    SPRING_APPLICATION_NAME: tzolov-data-flow-server
+    SPRING_PROFILES_ACTIVE: cloud
+    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{enabled: false}'
+    MAVEN_REMOTEREPOSITORIES[REPO1]_URL: https://repo.spring.io/libs-snapshot
+    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_URL: https://api.cf.violet.springapps.io
```

Comment:
Let's please remove any reference to real APIs.

Reply:
Sure i will, but i guess then we need to fix this too: https://dataflow.spring.io/docs/installation/cloudfoundry/cf-cli/#installing-using-a-manifest

Reply:
Good point. Huron was repaved/deleted months ago, though.